postgres/init/21-portal-basedata.sql

```diff
@@ -516,6 +561,61 @@ COPY portal.notification_type (id, label) FROM stdin;
 \.
 
 
+--
+-- Data for Name: service_licenses; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.service_licenses (id, license_text) FROM stdin;
+\.
+
+
+--
+-- Data for Name: service_statuses; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.service_statuses (id, label) FROM stdin;
+1	CREATED
+2	IN_REVIEW
+3	ACTIVE
+4	INACTIVE
+\.
+
+
+--
+-- Data for Name: services; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.services (id, date_created, name, thumbnail_url, provider_company_id, service_status_id, contact_email, sales_manager_id, last_editor_id) FROM stdin;
+\.
+
+
+--
+-- Data for Name: service_assigned_licenses; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.service_assigned_licenses (service_id, service_license_id) FROM stdin;
+\.
+
+
+--
+-- Data for Name: service_descriptions; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.service_descriptions (id, description, service_id, language_short_name) FROM stdin;
```

Comment:
same here, that looks for me like a content table instead of basedata

Reply:
moved as well